modules/dqt/ajax/saveQuery.php

```diff
@@ -33,6 +33,9 @@
     $couchConfig['adminpass']
 );
 $qid         = $user->getUserName() . "_" . $_REQUEST['QueryName'];
+if (preg_match('/\s/', $qid)) {
+    $qid = rawurlencode($qid);
+}
```

Comment:
is there a reason not to encode it all the time? why only encode it if there are white spaces ?